de-de/bc.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+language: bc
+contributors:
+    - ["caminsha", "https://github.com/caminsha"]
+filename: learnbc.bc
```

Comment:
Please change this to 
```
filename: learnbc-de.bc
```
Also add the following line after this 
```
lang:  de-de
```

Reply:
I opened a new pull request because I could not get the branch anymore. 

New Pull Request is [here](https://github.com/adambard/learnxinyminutes-docs/pull/3808)